cas-server-core/src/main/java/org/jasig/cas/monitor/AbstractPoolMonitor.java

```diff
@@ -123,7 +123,7 @@ public PoolStatus observe() {
          *
          * @param monitor the monitor
          */
-        public Validator(final AbstractPoolMonitor monitor) {
+        Validator(final AbstractPoolMonitor monitor) {
```

Comment:
Classes that are private dont need their ctor to be marked as public. That is redundant and useless.